nemo_curator/datasets/doc_dataset.py

```diff
@@ -178,6 +180,7 @@ def to_parquet(
         output_path: str,
         write_to_filename: Union[bool, str] = False,
         keep_filename_column: bool = False,
+        partition_on: Optional[str] = None,
     ):
         """
         See nemo_curator.utils.distributed_utils.write_to_disk docstring for parameters.
```

Comment:
Same comment as above.

Reply:
Added it here: https://github.com/NVIDIA/NeMo-Curator/pull/519/commits/d16047343f7413ee3f82df4fc4d38b032419e146